DSCResources/MSFT_xAdcsOnlineResponder/MSFT_xAdcsOnlineResponder.psm1

```diff
@@ -0,0 +1,95 @@
+#region Get Resource
```

Comment:
Can you add a comment with this link where you think is appropriate:

https://technet.microsoft.com/en-us/library/cc725958.aspx